ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml

```diff
@@ -663,6 +694,228 @@ objects:
           done
 
           echo "bootstrap: ${BOOTSTRAP_IP} control-plane: ${CONTROL_PLANE_0_IP} ${CONTROL_PLANE_1_IP} ${CONTROL_PLANE_2_IP} compute: ${COMPUTE_0_IP} ${COMPUTE_1_IP}"
+        elif [[ "${CLUSTER_TYPE}" == "gcp" ]]; then
+          mkdir -p /tmp/gcp
+
+          # Copy sample UPI files
+          cp -r /var/lib/openshift-install/upi/${CLUSTER_TYPE}/* /tmp/gcp
+
+          # Download, install, and configure gcloud
+          pushd /tmp
+          curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-256.0.0-linux-x86_64.tar.gz
+          tar -xzf google-cloud-sdk-256.0.0-linux-x86_64.tar.gz
+          export PATH=$PATH:/tmp/google-cloud-sdk/bin
+          mkdir gcloudconfig
+          export CLOUDSDK_CONFIG=/tmp/gcloudconfig
+          gcloud auth activate-service-account --key-file="${GOOGLE_CLOUD_KEYFILE_JSON}"
+          gcloud config set project openshift-gce-devel-ci
+          popd
+
+          export BASE_DOMAIN_ZONE_NAME="${GCP_BASE_DOMAIN_ZONE_NAME}"
```

Comment:
Hmm, not the biggest fan of this env on the pod.. can we keep this inline here or just query based on based_domain_dns_name

Reply:
Done. Pulled from gcloud.